Puppetfile

```diff
@@ -27,6 +27,10 @@ mod 'rtyler/jenkins',
 mod 'jhoblitt/lsststack',
   :git => 'https://github.com/lsst-sqre/puppet-lsststack.git',
   :ref => '2f413ee23997460d1ae218a431b9762ae26ac9f8'
+
+mod 'jhoblitt/ganglia', '~> 2.0'
```

Comment:
That is a publicly release dmodule that I have maintained for years.  It is also the puppetlabs blessed module for managing ganglia.  https://forge.puppetlabs.com/jhoblitt/ganglia